manifest.json

```diff
@@ -1,6 +1,21 @@
 {
-    "name": "Hello, world!",
+    "name": "Totally for real actual ghostery extension",
     "version":"0.000001",
-    "manifest_version": 2
+    "manifest_version": 2,
+    "content_scripts": [
+        {
+            "matches": ["<all_urls>"],
+            "js": ["content.js"]
+        }
+    ],
+    "browser_action": {
+        "default_icon": {
+            "16": "icon99.png",
+            "24": "icon99.png",
+            "32": "icon99.png"
+        },
+        "default_title": "gh05t3ry",
+        "default_popup": "popup.html"
+    }
 
 }
```

Comment:
FYI this icon is GitHub complaining that there's no newline at the end of the file. It's a good habit to make sure there always is one. Why you ask? Here's one good discusson: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Added newline to end of file